packages/internals/src/__tests__/getSchema.test.ts

```diff
@@ -62,7 +73,9 @@ it('reads from --schema args first even if package.json is provided', async () =
 it('throws if schema args path is invalid', async () => {
   const res = await testSchemaPath('pkg-json-with-schema-args', path.resolve(FIXTURE_CWD, 'wrong_path'))
 
-  expect(res).toMatchInlineSnapshot(`[Error: Provided --schema at ./__fixtures__/getSchema/wrong_path doesn't exist.]`)
+  expect(res).toMatchInlineSnapshot(
+    `[Error: Could not load --schema from provided path ../wrong_path: file or directory not found]`,
```

Comment:
New error message when explicitly provided `--schema` argument schema not found

Reply:
Nit, maybe:
```suggestion
    `[Error: Could not load \`--schema\` from provided path `\../wrong_path\`: file or directory not found]`,
```